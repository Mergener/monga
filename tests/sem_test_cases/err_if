function foo(x : int) {
    if (x < 5.0) {
        return;
    }
}