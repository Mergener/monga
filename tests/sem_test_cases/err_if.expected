[Error] Module sem_test_cases/err_if, line 2, column 17: Invalid comparison between values of types 'int' and 'double'.
Semantic errors were caught during analysis.
